app/src/main/java/fr/free/nrw/commons/category/CategoryDetailsActivity.java

```diff
@@ -181,7 +182,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
         // Handle item selection
         switch (item.getItemId()) {
             case R.id.menu_browser_current_category:
-                Utils.handleWebUrl(this, Uri.parse(Utils.getPageTitle(categoryName).getCanonicalUri()));
+                PageTitle title = Utils.getPageTitle(String.format("Category:%s", categoryName));
```

Comment:
I believe it's better to use the `CATEGORY_PREFIX` variable from `CategoryClient` here rather than hard coding the prefix.

Reply:
You're right, I have done that now.